### ISSUE TITLE
[React Version] Update Footer Social Media Icons

### ISSUE THREAD

**Issue by smaranjitghose** [smaranjitghose/awesome-portfolio-websites#1156]
## Instructions:

- Refer to the hosted version
-  All the src code would be in the `reactjs` branch

**Comment by anushbhatia** [comment]
Sure go ahead

**Comment by Shruti025** [comment]
hey, can I join this issue as well?

**Comment by smaranjitghose** [comment]
@Shruti025 I have asssinged you another issue, once that is done, you may take lead on this if it's incomplete by then.

**Comment by Lovakush** [comment]
@Shruti025 you completed or not?

**Comment by Shruti025** [comment]
hey @Lovakush I am working on the footer issue right now.

**Comment by lavakush07** [comment]
"I want to work on this issue under JWOC"

**Comment by raxvab** [comment]
I want to work on this issue under JWOC it's gonna be one of the first opportunities for me to start with open source can i get this issue assigned.

**Comment by Shruti025** [comment]
hey, is this issue still open? If yes then I can work on it under JWOC

**Comment by Shruti025** [comment]
hey, is this issue still unresolved? If yes, then I can work on it under JWOC

**Comment by smaranjitghose** [comment]
@Shruti025 updates?

**Comment by Shruti025** [comment]
hey, I wasn't able to run the project on my local device because of a recurring error. I conveyed the same on the discord group as well.

**Comment by Shruti025** [comment]
hey @smaranjitghose @anushbhatia the code is finally working! So I'll try my best to submit a PR by tomorrow.
